404.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: 404
+title: Page not found
+description: The page you requested could not be found.
+lang: en
+---
+
+# Not Found
```

Comment:
1. Could you use the title here?
2. I'd like the wording "Page Not Found" better than "Not Found."
3. Also use front matter liquid unless there is a reason not to.

`# {page.title} `

`{page.description}`

Reply:
sure

Reply:
I can't use liquid for the description because we are using link, so i'm going to leave it as it is.

Reply:
Oh really? Okay then.
I'd still ask for that change in the title wording unless you are against it.

Reply:
Also I misread. U did change the title. LGTM.